Jenkinsfile

```diff
@@ -5,7 +5,7 @@ pipeline {
         GIT_NAME = "volto-block-style"
         NAMESPACE = "@eeacms"
         SONARQUBE_TAGS = "volto.eea.europa.eu"
-        DEPENDENCIES = ""
+        DEPENDENCIES = "@eeacms/volto-resize-helper"
```

Comment:
@razvanMiu Not needed anymore if you correctly define dependencies within package.json